app/reducers/contactsform.js

```diff
@@ -201,6 +201,17 @@ contactFormSelectors.filteredNetworkNodes = createSelector(
   }
 )
 
+contactFormSelectors.isSearchValidNodeAddress = createSelector(
+  searchQuerySelector,
+  searchQuery => {
+    if (!searchQuery) {
+      return false
+    }
+    const [pubkey, host] = searchQuery.split('@')
```

Comment:
Could be a little more efficient if you first check for `searchQuery.length` to be equal 2

Reply:
I don't think that would have the same effect. I specifically want to ensure that we have something before and after an `@` to ensure that what we have looks like a node address.

Reply:
Yes, that's why I said first. So it's an addition. You don't need to split if length != 2 right?

Reply:
Got it. Updated.